### DISCUSSION TITLE
Malicious wiki edits

### DISCUSSION THREAD

**Discussion by Itxi** [opentrack/opentrack#1472] [General]
Hey all, someone edited the wiki front page to download some software renamed to make people think it was opentrack.

I've reverted their changes but I thought I should let people know.

I also stupidly ran the software first, I can't find any changes to my PC but I'll be honest I'm also not 100% certain what it was attempting to do. Looking at the files it looks like it was trying to send crypto, luckily I don't have any but I'm a little wigged out now tbh. If anyone could tell me what it was doing I'd appreciate it!

**Comment by Slion** [comment]
Scary stuff, we ought to track down who did the edit and have it ban. I think there are ways to avoid this kind of issues by limiting which accounts are allowed to modify the wiki.

**Reply by sthalik** [reply]
Most of wiki editing comes from people without repo commit rights. For now, let's remove the download link from the wiki since the user can easily navigate to it anyway.

@Itxi thank you for spotting this.

**Reply by GO63-samara** [reply]
It's horrible.
Looks like the link has changed a few times already.
For example, March 17, March 18 and May 15:
https://github.com/opentrack/opentrack/wiki/Home/_history

**Reply by Itxi** [reply]
I found the person who made the edit and reported them so hopefully their account will be removed, looking at the names of other files in the repo they directed to it seems they have tried to do this with several wikis.
